app/scripts/directives/podMetrics.js

```diff
@@ -130,6 +146,18 @@ angular.module('openshiftConsole')
         // Show last hour by default.
         scope.options.timeRange = _.head(scope.options.rangeOptions);
 
+        function generateVolumeDataSets() {
+          angular.forEach(scope.pod.spec.volumes, function(volumeData) {
+            if (volumeData.persistentVolumeClaim !== undefined) {
+              volumeDataSets.push({
+                id: "filesystem/usage",
```

Comment:
I fetch all metric definitions and found  "filesystem/usage" is also a container-level metric just like "cpu/usage" and "memery/usage". 
In my opinion, the volumeDataSets will include all persistent volumes. The volumes graph will display all pv metrics.